### DISCUSSION TITLE
Error in post-command-hook (vertico--exhibit): (cyclic-function-indirection ConTeXt-mode)

### DISCUSSION THREAD

**Discussion by WorldsEndless** [minad/vertico#475] [Q&A] [answered]
Having upgraded everything, now my Vertico (read: most selection commands, including find-file and `switch-to-buffer` seem to throw the error, `Error in post-command-hook (vertico--exhibit): (cyclic-function-indirection ConTeXt-mode)` at least the first time they are called. It's beyond troublesome to be unable to use `m-x` without the error,

```
Error in post-command-hook (vertico--exhibit): (cyclic-function-indirection ConTeXt-mode)
```

When I look up the library, I see that Context is provided by AucTex, which did receive a massive upgrade the last time around. Does anyone know a solution to this issue?

**Comment by WorldsEndless** [comment, marked as answer]
I "solved" this by disabling all tex things and not loading them at startup. The error went away; fortunately I am not currently doing any latex work. I was running auctex 9058b58, updated at the end of April 2024. I'll see if I can submit an error to them (though their debugging process predates git so... it's a thing)

**Comment by YasserKa** [comment]
For TeX users. You can use

`(fmakunbound 'ConTeXt-mode)`
